newrelic_python_agent/plugins/rabbitmq.py

```diff
@@ -20,6 +20,7 @@ class RabbitMQ(base.Plugin):
     DEFAULT_HOST = 'localhost'
     DEFAULT_PORT = 80
     DEFAULT_API_PATH = '/api'
+    DEFAULT_TIMEOUT = (3.05, 30)
```

Comment:
Perhaps a comment explaining that it's the connect timeout followed by the read timeout would be nice :)
And optionally a link to http://docs.python-requests.org/en/master/user/advanced/#timeouts as well